setup.py

```diff
@@ -7,7 +7,7 @@
 setup(
     name='potassium',
     packages=['potassium'],
-    version='0.2.0',
+    version='0.2.2',
```

Comment:
according to semantic versioning i think this should be a minor version bump. ie `0.3.0` since it adds to the API

Reply:
Good point.